src/steps/index.js

```diff
@@ -26,5 +27,6 @@ export const steps = [
 ]
 
 export const nextSteps = {
-  roof: roofNextSteps
+  roof: roofNextSteps,
+  'exterior-paint': extPaintNextSteps
```

Comment:
we have this strange thing that we do: we use strings as keys in objects just because we want to write in `dash-case`, but we can write in `camelCase` and it would be lot easier everywhere in the application.